### ISSUE TITLE
What is the meaning of warmup_steps when I fine-tune the model, can I remove it?

### ISSUE THREAD

**Issue by wmathor** [UKPLab/sentence-transformers#497]
```python
evaluator = evaluation.EmbeddingSimilarityEvaluator(sentences1, sentences2, scores)
# Define your train dataset, the dataloader and the train loss
train_dataset = SentencesDataset(train_data, model)
train_dataloader = DataLoader(train_dataset, shuffle=True, batch_size=32)
train_loss = losses.CosineSimilarityLoss(model)

# Tune the model
model.fit(train_objectives=[(train_dataloader, train_loss)], epochs=1, warmup_steps=100, evaluator=evaluator, evaluation_steps=100, output_path='./Ko2CnModel')
```

**Comment by nreimers** [comment]
See the original BERT paper on it.

As you fine-tune a model, and not train it from scratch, warm-up is recommended to not "destroy" what the model has previously learned.

**Comment by wmathor** [comment]
ok, Maybe I know what this means.
Thank you every much
